"robot_ip_address": "192.168.0.2"
"robot_port": 5007
"robot_speed": 20.0
"thread_sampling_time_nsec": 4000000
"q_min": [1.91986218, 1.57079633, 2.35619449, 0.78539816, 0.08726646, 1.57079633]
"q_max": [-1.91986218, 0.17453293, 0.78539816, -0.78539816, -1.57079633, -1.57079633]
